locals.tf

```diff
@@ -4,6 +4,14 @@ locals {
     (contains(["rapid", "stable", "node-image"], var.automatic_upgrade_channel) && var.kubernetes_version == null && var.default_node_pool.orchestrator_version == null)
   )
   dns_prefix = coalesce(var.dns_prefix, random_string.dns_prefix.result)
+  kube_admin_enabled = (
+    !var.local_account_disabled
+    ? (
+      var.azure_active_directory_role_based_access_control != null &&
```

Comment:
I don't think we need this null check since a null panic would be captured by `try`